src/main/java/io/ebeaninternal/server/type/ScalarTypeFile.java

```diff
@@ -194,13 +197,19 @@ public long pump(InputStream is, OutputStream out) throws IOException {
     } finally {
       if (output != null) {
         try {
+          // Closing BufferedOutputStream does not close the underlying OutputStream.
+          // It only performs a flush() on the underlying OutputStream.
```

Comment:
Thats not true, close method of FilterOutputStream
```java
    public void close() throws IOException {
        try (OutputStream ostream = out) {
            flush();
        }
    }
```
(the .close is a bit hard to find... Note that there is a try-with-resources statement)